code/script.js

```diff
@@ -1,9 +1,159 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
+const form = document.getElementById('name-form');
+let value = document.getElementById('input-value').value;
+const inputWrapper = document.getElementById("input-wrapper")
 
-// Global variables, if you need any, declared here
+//Question 1 - location
+const askForLocation = () => {
+  showMessage (`I am here to help you chose your next holiday destination. Let me know what view you prefer...`, "bot")
+  inputWrapper.innerHTML= `
+    <select id= "select">
+      <option id= "" selected disabled> Chose from this list</option>
+      <option value="mountain" id="mountainOption">Mountain</option>
+      <option value="city" id="cityOption">City</option>
+      <option value="beach" id="beachOption">Beach</option>
+    </select>
+  `
+const select = document.getElementById("select")
+  select.addEventListener("change", () => {
+  if (select.value === "mountain") {
+      showMessage ("I wanna go all the way to the top!", "user")
+      setTimeout (() => askForWarmOrColdMountain (), 2000) 
+  } else if (select.value === "city") {
+      showMessage ("I wanna spend money!", "user")
+      setTimeout(()=> askForWarmOrColdCity(), 2000)
+  } else {
+      showMessage ("I want to relax in the beach", "user")
+      setTimeout(() => askForWarmOrColdBeach (), 2000)
+    } 
+ })
+}
+
+// Question 2 - warm or cold
+const askForWarmOrColdMountain = () => {
+  showMessage (`We can make that happen! Now, should there be cold or warm in the top?`, "bot")
+  inputWrapper.innerHTML=`
+    <button value= "warm" class="warm-button" id="warm-button">Warm</button>
+    <button value= "cold" class="cold-button" id="cold-button">Cold</button>
+  `
+  document.getElementById("warm-button").addEventListener("click", () => {
+    showMessage ("I prefer to be hot as the sun.", "user")
+    setTimeout (() => showResultMountainWarm(), 1000)
+  })
+
+  document.getElementById("cold-button").addEventListener("click", () =>{
+    showMessage ("I prefer to be cold as an icebear.", "user")
+    setTimeout (() => showResultMountainCold(), 1000)
+  })
+}
```

Comment:
This code structure is repeated 3 times what is the ideal situation to create a more generic function to handle this task every time 🤓 Please try to do it 😸 If you will you can always ask me for more tips 😸